File: 605 Homeworks/Submitted SRCs/HW2/src/game/Player.java
package game;

import java.lang.reflect.Array;

/**
 * This class is an enumeration for representing the players in the Dots And
 * Boxes game.
 *
 * @author Tyler Ronnenberg
 * @author Liam Cui
 */
public enum Player {
    /**
     * Enum constants are the Owner types available; player BLUE and RED as
     * well as NONE for when no owner is assigned.
     */
    BLUE,
    NONE,
    RED;

    /**
     * Gets the player's label
     * @return A string containing the player's label
     */
    public String getLabel() {
        if (this == BLUE) {
            return "B";
        } else if (this == RED) {
            return "R";
        } else {
            return " ";
        }
    }

}

File: Islands/lab9/src/player.py
"""
CSCI-603 Islands Lab
Author: Liam Cui

An enum representing who owns a cell. RED and BLUE represent the players while
NONE represents no ownership.
"""
import enum
class Role(enum.Enum):
    NONE = 'NONE'
    RED = 'RED'
    BLUE = 'BLUE'
    def __str__(self):
        return self.value